client/src/app/components/home/home.component.html

```diff
@@ -111,6 +111,28 @@
   </div>
 </div>
 
+<div class="sec-3">
+  <div class="container">
+    <div class="d-md-flex align-items-center justify-content-between-w-100">
```

Comment:
create a div when up to medium size screens are flex and small size screens are block

Reply:
done